rendertron depends on semver, but semver is not in the package.json

$ npm install -g rendertron
$ rendertron
module.js:538
    throw err;
    ^

Error: Cannot find module 'semver'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/rendertron/bin/rendertron:7:14)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)

--- a/package.json
+++ b/package.json
@@ -27,6 +27,7 @@
     "express": "^4.15.2",
     "performance-now": "^2.1.0",
     "progress-bar-element": "^2.0.1",
+    "semver": "^5.4.1",
     "uuid": "^3.1.0"
   },
   "devDependencies": {

